src/v/redpanda/application.cc

```diff
@@ -529,7 +529,8 @@ void application::configure_admin_server() {
       std::ref(shard_table),
       std::ref(metadata_cache),
       std::ref(archival_scheduler),
-      std::ref(_connection_cache))
+      std::ref(_connection_cache),
+      node_status_provider.get())
```

Comment:
i don't think this commit will compile since `node_status_provider` isn't added until a later commit.

Reply:
Fixed.